sky/adaptors/vast.py

```diff
@@ -0,0 +1,29 @@
+"""Vast cloud adaptor."""
+
+import functools
+
+_vast_sdk = None
+
+
+def import_package(func):
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        global _vast_sdk
+
+        if _vast_sdk is None:
+            try:
+                import vastai_sdk as _vast  # pylint: disable=import-outside-toplevel
+                _vast_sdk = _vast.VastAI()
+            except ImportError:
+                raise ImportError('Fail to import dependencies for vast.'
+                                  'Try pip install "skypilot[vast]"') from None
```

Comment:
Should we also update the setup.py?

https://github.com/skypilot-org/skypilot/blob/ed4329a724aa583dd02e325e824fd1e36ee32aaf/sky/setup_files/setup.py#L211-L250

Reply:
oh this is where that is ... yes. I've released the sdk ... I'll do that.

Reply:
it's at 292744972424d8a8cf1a8d6814d67c0b952f1ecb